configure.ac

```diff
@@ -3450,7 +3487,7 @@ PAC_SET_MPI_TYPE(01, MPI_CHAR, 1)
 PAC_SET_MPI_TYPE(02, MPI_UNSIGNED_CHAR, 1)
 PAC_SET_MPI_TYPE(03, MPI_SHORT, $ac_cv_sizeof_short)
 PAC_SET_MPI_TYPE(04, MPI_UNSIGNED_SHORT, $ac_cv_sizeof_short)
-PAC_SET_MPI_TYPE(05, MPI_INT, $ac_cv_sizeof_int)
+AC_DEFINE_UNQUOTED([MPI_INT_INTERNAL], [MPIR_SIGNED_INT$ac_cv_sizeof_int], [Internal type for MPI_INT])
```

Comment:
Maybe you should use the MPIR_ namespace and name the internal type `MPIR_INT_INTERNAL` ? It is a bit confusing seeing the MPI_ namespace used with something is not really in MPI.

Reply:
Sure.